lib/archethic/reward.ex

```diff
@@ -28,6 +35,21 @@ defmodule Archethic.Reward do
 
   @doc """
   Create a transaction for minting new rewards
+
+  ## Examples
+
+    iex> case Reward.new_rewards_mint(2_000_000_000) do
```

Comment:
In the doctest you directly put the expected result, so no need to make the check if's working on not in

```elixir
iex> Reward.new_reward_mint(2_000_000_000)
%Transaction{ 
  type: :mint_rewards,
  data: %TransactionData{
      content: "..."
  }
}
```